test/main.yml

```diff
@@ -68,6 +68,11 @@
                       - postgis
                       - postgis_topology
                       - btree_gist
+                schemas:
+                  - description: A readonly schema
+                    name: readonly
+                    default_privileges:
```

Comment:
Could you just add a test with no default_privileges, I see it as something which can break in the future